app/controllers/reviews_controller.rb

```diff
@@ -118,4 +119,39 @@ def check_review_period
 
     redirect_to root_path, alert: t('reviews.edit.errors.conference_out_of_range')
   end
+
+  def create_third_reject(review)
+    return if in_early_review_phase?
+
+    return unless review.weak_reject? || review.strong_reject?
+
+    session_reviews = review.session.early_reviews + review.session.final_reviews
```

Comment:
Why do `session_reviews` count `early_reviews`? And, also, why do `other_reviews` don't?

Reply:
They shouldn't. Early reviews were meant to represent only mid-submission period feedback to the authors and were not used to decide the final state. We should remove this from the condition.